### DISCUSSION TITLE
Sorting fields in alerts doesn't persist

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#8126] [Unsupported Versions] [answered]
Hi so updated to 2.3.130 and I've noticed in the alerts that the sorting doesn't persist through refreshes or coming back to the page etc, I normally sort by `event.severity_label` but now it seems to default to the `Count`. While it's not the end of the world it was a nice thing to have and I'm not sure if this is a bug or intentional going forward.

**Comment by xfaith** [comment]
Doug answered in this comment:  https://github.com/Security-Onion-Solutions/securityonion/discussions/8091

**Comment by dougburks** [comment, marked as answer]
This will be fixed in the upcoming 2.3.140 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/8133
